package/boot/uboot-rockchip/src/arch/arm/dts/rk3568-opc-h68k.dts

```diff
@@ -0,0 +1,277 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+#include "rk3568-evb.dts"
```

Comment:
No. You should use the same dts file as for the kernel (with rng node remove).

Never include evb dts, they're totally different boards.